decidim/decidim
# Redesign: add admin sidebar

#### :tophat: What? Why?
This PR adds the Admin sidebar to ease up the navigation, as discussed with @carolromero  and @andreslucena.

#### Testing
1. Login as admin and visit admin panel 
2. Visit single participatory process, and see there is a sidebar 
3. repeat the same for Assemblies, Initiatives, Votings, Conferences, Templates, Participants, Global moderations 

### :camera: Screenshots

Templates 
![image](https://github.com/decidim/decidim/assets/105683/650aa4e3-4694-41d4-b8ab-bfdf9882e407)

Settings ( already there ) 
![image](https://github.com/decidim/decidim/assets/105683/bc80ee08-c1f2-4ac2-a953-191ceb4d5e7b)

Participants 
![image](https://github.com/decidim/decidim/assets/105683/2c05e50e-ec56-466e-8717-e62ea75e005f)

Global moderations
![image](https://github.com/decidim/decidim/assets/105683/2f3dffe4-5228-42d7-aeb7-e3c26e39d2a4)

Conferences
![image](https://github.com/decidim/decidim/assets/105683/fccb857c-f3f8-4be7-9cbd-304b0cd0af96)

Votings
![image](https://github.com/decidim/decidim/assets/105683/d5dd188d-c599-473a-aac6-ca14959cbecf)

Initiatives
![image](https://github.com/decidim/decidim/assets/105683/88948aa3-2b39-42d6-9e08-e65644138e69)

Assemblies 
![image](https://github.com/decidim/decidim/assets/105683/3e187a4c-ae50-4b47-a9de-2efe65e14e0b)

Participatory process: 
![image](https://github.com/decidim/decidim/assets/105683/8d0a0668-a9f1-492e-8f10-6aa07b994a97)

Participatory process manage menu with components listed: 
![Screenshot from 2023-09-26 23-11-41](https://github.com/decidim/decidim/assets/105683/50b9f57b-5852-4add-97d0-d9afa9fd2c90)

Participatory process manage menu without components listed: 
![Screenshot from 2023-09-26 23-16-18](https://github.com/decidim/decidim/assets/105683/26700242-13f7-410e-a46e-eae8f189ff02)




:hearts: Thank you!
